experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -142,12 +137,12 @@ export const isIgnored = (
  * Sets the span with the error passed in params
  * @param {Span} span the span that need to be set
  * @param {Error} error error that will be set to span
- * @param {(IncomingMessage | ClientRequest)} [obj] used for enriching the status by checking the statusCode.
+ * @param {SpanStatusCode} [code] used for overriding the status
  */
 export const setSpanWithError = (
   span: Span,
   error: Err,
-  obj?: IncomingMessage | ClientRequest
+  code = SpanStatusCode.ERROR
```

Comment:
If there is no specific reason to make this optional/default I would prefer a required param. Calling with no status code will always result in an error anyway so reading code at call sites will be more obvious if they always include that.

Reply:
It was only used in one place. I've removed the parameter, and changed that callsite to use `span.setStatus` directly.